src/app-gocardless/services/gocardless-service.js

```diff
@@ -211,6 +212,20 @@ export const goCardlessService = {
       goCardlessService.getBalances(accountId),
     ]);
 
+    transactions.transactions.booked = transactions.transactions.booked.map(
+      (t) => ({
+        ...t,
+        payeeName: formatPayeeName(t),
+      }),
+    );
+
+    transactions.transactions.pending = transactions.transactions.pending.map(
+      (t) => ({
+        ...t,
+        payeeName: formatPayeeName(t),
+      }),
+    );
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏instead of doing this uniformly for ALL banks the same way - I think we should allow the bank-providers to do their own payee normalisation (if they wish to).

So my suggestion here is to move this payee name normalisation inside [here](https://github.com/actualbudget/actual-server/blob/1da9c821ee858a38f9bbc11d4949a398a44c989a/src/app-gocardless/banks/integration-bank.js#L48).

Other than that - LGTM!

Reply:
That was my initial approach but I turned away from it because this way replicates the behaviour prior to this change
If needed creditorName/debtorName can be altered in the bank handler to change the payee name, but I think that the default behaviour to fall back on other fields and append the IBAN when present is best retained.

Let me know what you reckon, happy to change if you disagree.

Reply:
We can still keep the same behavior as before. But just have the normalization logic in a different location (the generic bank handler + the overrides). 

IMO that would be more future proof solution since it would allow to change this logic on a per-bank basis.